pkg/blobinfocache/internal/test/test.go

```diff
@@ -215,8 +223,8 @@ func testGenericCandidateLocations2(t *testing.T, cache blobinfocache.BlobInfoCa
 		scope := types.BICTransportScope{Opaque: scopeName}
 
 		// Nothing is known.
-		assert.Equal(t, []blobinfocache.BICReplacementCandidate2{}, cache.CandidateLocations2(transport, scope, digestUnknown, false))
-		assert.Equal(t, []blobinfocache.BICReplacementCandidate2{}, cache.CandidateLocations2(transport, scope, digestUnknown, true))
+		assert.Equal(t, []blobinfocache.BICReplacementCandidate2{blobinfocache.BICReplacementCandidate2{Digest: digestUnknown, CompressorName: blobinfocache.UnknownCompression, UnknownLocation: true, Location: types.BICLocationReference{Opaque: ""}}}, cache.CandidateLocations2(transport, scope, digestUnknown, false))
+		assert.Equal(t, []blobinfocache.BICReplacementCandidate2{blobinfocache.BICReplacementCandidate2{Digest: digestUnknown, CompressorName: blobinfocache.UnknownCompression, UnknownLocation: true, Location: types.BICLocationReference{Opaque: ""}}}, cache.CandidateLocations2(transport, scope, digestUnknown, true))
```

Comment:
I thought the v2 API never returns `UnknownCompression`??!

Reply:
This change is not needed, my bad.